src/main/resources/db/changelog/changes/create-role-table.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-role-table
+      author: YEvhenii
+      changes:
+        - createTable:
+            tableName: role
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+
```

Comment:
Redundant line. Remove in all files

Reply:
Empty line should be at the end of file, shouldn't she?

Reply:
Yeah, but it seems, that you have 2 lines 🤔